code/modules/cargo/console.dm

```diff
@@ -62,14 +66,76 @@
 	obj_flags |= EMAGGED
 	contraband = TRUE
 
+	permits |= PREMIT_SPACE
+	permits |= PREMIT_WEAPONS
+	permits |= PERMIT_MEDCO
+	permits |= PERMIT_BLACKMARKET
+	permits |= PERMIT_ANIMALS
+	permits |= PERMIT_ADVSCI
+
 	// This also permamently sets this on the circuit board
 	var/obj/item/circuitboard/computer/cargo/board = circuit
+
 	board.contraband = TRUE
+
+	board.permits |= PREMIT_SPACE
+	board.permits |= PREMIT_WEAPONS
+	board.permits |= PERMIT_MEDCO
+	board.permits |= PERMIT_BLACKMARKET
+	board.permits |= PERMIT_ANIMALS
+	board.permits |= PERMIT_ADVSCI
+
 	board.obj_flags |= EMAGGED
 	req_access = list()
 	update_static_data(user)
 	return ..()
 
+/obj/machinery/computer/cargo/attackby(obj/item/W, mob/living/user, params)
+	var/obj/item/circuitboard/computer/cargo/board = circuit
+	if((istype(W, /obj/item/folder/permit/space_gear)) && allowed(user) && !CHECK_BITFIELD(permits,PREMIT_SPACE))
```

Comment:
Please standarize the permit folders behaviour, By also storing their related permit flags and supply shuttle added callTime in new variables. This is a mess.

Reply:
what?

Reply:
Don't make a huge "else if" chain of copypasted code, it goes against the coding standards and guidelines for this project.

Reply:
I was told to do it this way... By a maintainer do to it working...